src/cider/nrepl.clj

```diff
@@ -437,7 +437,11 @@ Depending on the type of the return value of the evaluation this middleware may
     :expects #{}
     :handles {"stacktrace" {:doc "Return messages describing each cause and stack frame of the most recent exception."
                             :optional wrap-print-optional-arguments
-                            :returns {"status" "\"done\", or \"no-error\" if `*e` is nil"}}}}))
+                            :returns {"status" "\"done\", or \"no-error\" if `*e` is nil"}}
+              "parse-stacktrace" {:doc "Parse the `stacktrace` and return messages describing each cause and stack frame."
```

Comment:
@bbatsov I added a new op called `parse-stacktrace`. Are you fine with this name?

Reply:
Hmm, I think I rename it to `analyze-stacktrace` ...